app/routes/api/v1/ecommerce/customers/index.ts

```diff
@@ -25,16 +26,18 @@ export const action: ActionFunction = async ({ request }) => {
 
 export const postRequest = async (request: Request): Promise<Response> => {
   const schema = z.object({
-    email: z.string(),
-    password: z.string(),
-    firstName: z.string(),
-    lastName: z.string(),
+    email: z.string().email(),
+    password: z.string().min(1).max(16),
+    firstName: z.string().min(1).max(25),
+    lastName: z.string().min(1).max(25),
```

Comment:
Again, it is not a good practice to limit the hash, also for `firstName` and `lastName`.

There is no a need to limit them for the DB.

Reply:
@nullndr you should have listened to Arciero's lesson. If the field is a name, it probably isn't longer than maybe 50 char. The character limitation is good to avoid dumping junk into the db